transforms/language/pdf2parquet/python/src/pdf2parquet_transform.py

```diff
@@ -114,7 +112,7 @@ def _convert_pdf2parquet(self, doc_filename: str, ext: str, content_bytes: bytes
         if self.contents_type == pdf2parquet_contents_types.MARKDOWN:
             content_string = doc.render_as_markdown()
         elif self.contents_type == pdf2parquet_contents_types.JSON:
-            content_string = pd.io.json.ujson_dumps(doc.render_as_dict(), double_precision=2)
+            content_string = pd.io.json.ujson_dumps(doc.render_as_dict(), double_precision=0)
```

Comment:
are these scores less than 1 that are being truncated to 0 precision after the decimal?  Do you still need this?

Reply:
The JSON format contains bbox. It seems that the precision on the CI workers is very unpredictable, so this is rounding them to integer.